app/models/user.rb

```diff
@@ -51,9 +51,21 @@ def reset_password!(password)
     save!
   end
 
+  def email_activate
```

Comment:
It's better to use ! in the end of the method. This method changes the object.

Reply:
Fixed.